src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts

```diff
@@ -293,7 +307,7 @@ describe('Detailed view and edit a selected work item - ', () => {
     comp.loggedIn = fakeAuthService.isLoggedIn();
```

Comment:
ugh, so I tried the change and then undid the fix locally and I don't see the Type error anymore. Please at least take a look at it.